packages/filebrowser/src/listing.ts

```diff
@@ -806,6 +809,91 @@ export class DirListing extends Widget {
     }
   }
 
+  // Update the modified column's size
+  updateModifiedSize(node: HTMLElement) {
+    const modified = DOMUtils.findElement(node, ITEM_MODIFIED_CLASS);
+    this._modifiedWidth = modified.getBoundingClientRect().width;
```

Comment:
Test failure rightly highlights that modified can be `null`.

```suggestion
    this._modifiedWidth = modified ? modified.getBoundingClientRect().width : 200;
```

(or some other default)